spec/std/http/request_spec.cr

```diff
@@ -215,6 +215,11 @@ module HTTP
         request.should eq HTTP::Status::BAD_REQUEST
       end
 
+      it "is stores normalized case for common header name (#8060)" do
+        request = Request.from_io(IO::Memory.new("GET / HTTP/1.1\r\nconetnt-type: foo\r\n\r\n")).as(Request)
```

Comment:
```suggestion
        request = Request.from_io(IO::Memory.new("GET / HTTP/1.1\r\ncontent-type: foo\r\n\r\n")).as(Request)
```

Reply:
Thanks! I think I was running a different, spec, this one actually fails 😊